CHANGELOG.md

```diff
@@ -3,11 +3,116 @@
+
+Below we highlight some key changes to the defaults in OPA 1.0:
+
+- The `if` & `contains` keywords are now mandatory when declaring rule heads, not just when using the `rego.v1` import.
+- Other new keywords (`every`, `in`) are available without any imports.
+- Previously rules reserved for `opa check --strict` are now the default. Duplicate imports and imports which shadow each other are no longer allowed.
+- OPA 1.0 comes with a range of backwards compatibility features to aid your migrations, please see the [v0 compatibility guide](https://www.openpolicyagent.org/docs/edge/v0-compatibility/)
+if you must continue to support v0 Rego.
+
+Read more about the OPA 1.0 announcement on the [OPA blog](https://blog.openpolicyagent.org/).
+
+Following are other changes that are included in OPA 1.0.
+
+### Improvements to memory allocations
+
+PRs [#7172](https://github.com/open-policy-agent/opa/pull/7172), [#7190](https://github.com/open-policy-agent/opa/pull/7190),
+[#7193](https://github.com/open-policy-agent/opa/pull/7193), [#7165](https://github.com/open-policy-agent/opa/pull/7165),
+[#7168](https://github.com/open-policy-agent/opa/pull/7168), [#7191](https://github.com/open-policy-agent/opa/pull/7191) &
+[#7222](https://github.com/open-policy-agent/opa/pull/7222) together improve the memory performance of OPA. Key strategies
+include reusing pointers and optimizing array and object operations, minimizing intermediate object creation, and using `sync.Pool` 
+to manage memory-heavy operations. These changes cumulatively greatly reduced the number of allocations and improved
+evaluation speed by 10-20%. Additional benchmarks highlighted significant memory and speed improvements in custom
+function evaluation.
+
+Authored by @anderseknert.
+
+### Wrap http.RoundTripper for SDK users
+
+PR [#7180](https://github.com/open-policy-agent/opa/pull/7180) adds an `EvalHTTPRoundTrip` EvalOption and query-level `WithHTTPRoundTrip` option.
+Both use a new function type which converts an `http.Transport` configured by topdown to an `http.RoundTripper`.
+This supports use cases requiring the customization of the `http.send` built in behavior.
+
+Authored by @evankanderson.
+
+### Improvements to scientific notation parsing in `units.parse`
+
+PR [#7147](https://github.com/open-policy-agent/opa/pull/7147) extends the behaviour of `extractNumAndUnit` to support
+scientific notation values. This means values such as `1e3KB` can now be handled by this function.
+
+Authored by @berdanA.
+
+### Support customized buckets `bundle_loading_duration_ns` metric
+
+PR [#7156](https://github.com/open-policy-agent/opa/pull/7156) extends OPA’s Prometheus configuration to allow the
+setting of user defined buckets for metrics. This aids when debugging the loading of slow bundles.
+
+Authored by @jwu730-1.
+
+### Test suite performance improvements
+PR [#7126](https://github.com/open-policy-agent/opa/pull/7126) updates tests to improve performance. Topdown and `storage/disk/`
```

Comment:
[nit] Merely for consistency, it doesn't change the markup, let's add that newline:
```suggestion
### Test suite performance improvements

PR [#7126](https://github.com/open-policy-agent/opa/pull/7126) updates tests to improve performance. Topdown and `storage/disk/`
```